style.css

```diff
@@ -583,435 +932,645 @@ fieldset {
     }
   }
 
-  .logo:hover {
+  .animation-rotate {
     animation: rotate 0.5s linear 1s;
   }
 
-  .logo-menu nav {
+  @keyframes slide {
+    0% {
+      transform: translate(750px);
+    }
+
+    15% {
+      transform: perspective(-200px) translateZ(-50px);
+      color: #838ffc;
+    }
+
+    25% {
+      transform: perspective(-200px) translateZ(-100px);
+      color: #aab3ff;
+    }
+
+    50% {
+      transform: perspective(-200px) translateZ(-150px);
+      color: #d3d7fc;
+    }
+
+    100% {
+      transform: scale(1.2);
+      color: #ebedff;
+    }
+  }
+
+  .animation-slide {
+    animation: slide 0.7s ease-out 1.3s;
+  }
+
+  /* ---------- */
+
+  .nav-links {
     display: none;
   }
 
-  .header-desktop {
+  .nav-bar {
+    list-style: none;
     display: flex;
-    justify-content: center;
+    flex-direction: row;
     align-items: center;
-    margin-right: 9.063rem;
-    font-family: "Poppins", Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 0.938;
-    line-height: 20px;
-    color: #344563;
-    width: 259px;
-    height: 40px;
+    margin-top: 4.5rem;
+  }
+
+  .nav-bar li {
+    position: relative;
+    top: -1.2rem;
+    padding-right: 6rem;
+  }
+
+  .nav-bar li a {
+    text-decoration: none;
+    font-size: 1.5rem;
+    color: var(--text-color);
+    line-height: 2rem;
   }
 
-  .header-desktop p {
+  nav .fa {
+    display: none;
+  }
+
+  .headline-section {
+    height: 90vh;
+    background: var(--body-color) url(./images/HeaderBgDesktop.svg);
+    background-position: center;
+    background-size: 100% 99%;
+    border-bottom-left-radius: 175px;
+    width: 118vw;
+  }
+
+  .headline-section h1 {
+    margin-left: 40rem;
+    top: -2rem;
+  }
+
+  .headline-section p {
+    margin-left: 40rem;
+    width: 45%;
+    top: -2rem;
+    letter-spacing: 0.03rem;
+    font-size: 2rem;
+    transition: transform 0.5s;
+  }
+
+  .header-icons {
+    margin-left: 37.5rem;
+    top: -2rem;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-items: center;
+    margin-top: 10rem;
+    margin-left: 20%;
+  }
+
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- [ ] Make sure the `See Livesee  See Code` buttons are centered on the center of the page on the mobile size.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/217685671-99fc8b9f-4519-4d34-b754-9c4a0699d93f.png) |![Screenshot from 2022-10-12 05-00-43](https://user-images.githubusercontent.com/79058364/195239977-37fb4625-af99-43e0-8509-0edd240c0d50.png)
 


- [ ]  Please prevent your `See Livesee  See Code` buttons to overlap the text description on the desktop size and make sure you use the same margin-top or padding-top, I can see an extra space on the top  👨🏻‍🎨 🎨 

- Actually, Your popup desktop does not really match the Figma design. Please improve your design 🎨 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/217685581-963ee9f5-5afa-4c0a-8eb6-e78b5ce811b2.png)| ![Screenshot from 2022-10-12 05-13-25](https://user-images.githubusercontent.com/79058364/195241407-f696f0f5-78d1-4853-b3ea-baeebab49b3d.png)